dbbackup/management/commands/_base.py

```diff
@@ -0,0 +1,18 @@
+from optparse import make_option
+from django.core.management.base import BaseCommand, LabelCommand
+
+
+class BaseDbBackupCommand(LabelCommand):
```

Comment:
Great idea to inherit from label!

Remember though that setting these two are part of how that works?

    args = '<label label ...>'
    label = 'label'

...and wouldn't that break the API?